components/steering_wheel/SConscript

```diff
@@ -241,19 +241,38 @@ env.Alias("cdb", env.CompilationDatabase())
 env.Alias("upload", env.flash(source="build/steering_wheel.bin"))
 env.Alias("openocd", env.openocd_srv(interface=OPENOCD_INTERFACE, board=OPENOCD_BOARD))
 
+# give support for halting on gdb/openocd start
+mon_reset_cmd = '-ex "monitor reset'
+if GetOption("halt"):
+    mon_reset_cmd += 'halt"'
+else:
+    mon_reset_cmd += '"'
+
+env.Alias(
+    "flash-debug",
```

Comment:
Flashes the MCU without restarting and then loads gdb with MCU halted

Reply:
do we need this _and_ the `--halt` arg? Feels like just this is sufficient? Also I think it should be called `openocd-gdb-halt` since it does the same thing as `openocd-gdb` but also halts the chip